.github/workflows/release_examples.yml

```diff
@@ -20,41 +20,34 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
-      - name: Serialize Examples
-        env:
-          REGISTRY: "ghcr.io/flyteorg"
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: unionai/flytectl-setup-action@v0.0.1
+      - name: Start Flytectl sandbox
+        run: make setup
+      - name: Setup Flytectl config
+        run: flytectl config init
+      - name: Setup python requirments
         run: |
-          REGISTRY=ghcr.io/flyteorg VERSION=latest make -C cookbook/${{ matrix.directory }} serialize
-      - name: Archive Serialized protos
-        env:
-          REGISTRY: "ghcr.io/flyteorg"
+          mkdir release
+          pip install flytekit==0.20.0
+      - name: Test Serialize & Register ${{ matrix.directory }} example
+        if: github.event_name == 'pull_request'
         run: |
-          mkdir -p cookbook/release-snacks
-          make -C cookbook archive
-      - name: Run GoReleaser
+          OUTPUT_DIR=$(pwd)/release/ VERSION=latest make -C cookbook/${{ matrix.directory }} register
+          OUTPUT_DIR=$(pwd)/release/ VERSION=latest make -C cookbook/${{ matrix.directory }} fast_register
+      - name: Serialize ${{ matrix.directory }} example for release
         if: github.event_name != 'pull_request'
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          version: latest
-          args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.FLYTE_BOT_PAT }}
-
-  release-manifest:
-    name: Release Manifest
-    if: github.event_name != 'pull_request'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
-      - name: Serialize Examples
-        env:
-          REGISTRY: "ghcr.io/flyteorg"
+          REGISTRY: "cr.flyte.org/flyteorg"
         run: |
-          mkdir -p cookbook/release-snacks
-          cp cookbook/flyte_tests_manifest.json cookbook/release-snacks/flyte_tests_manifest.json
-      - name: Release test manifest
+          OUTPUT_DIR=$(pwd)/release/ VERSION=latest make -C cookbook/${{ matrix.directory }} serialize
+      - name: Clean Git
```

Comment:
Can we add a comment for why this is necessary?  Also, which file(s) are we removing?  If it's just one file can we `rm` the file instead?

Reply:
no we have more file changes when we install the requirements